packages/viewer/src/lib/volume-intersection/interactions.ts

```diff
@@ -1,51 +1,104 @@
-import { Point } from '@vertexvis/geometry';
+import { Disposable } from '@vertexvis/utils';
 
+import { boxQueryCursor } from '../cursors';
+import { getMouseClientPosition } from '../dom';
+import { InteractionApi } from '../interactions';
 import { InteractionHandler } from '../interactions/interactionHandler';
 import { VolumeIntersectionQueryController } from './controller';
 
 export class VolumeIntersectionQueryInteractionHandler
   implements InteractionHandler
 {
   private element?: HTMLElement;
+  private api?: InteractionApi;
   private isInteracting?: boolean;
+  private enabled?: boolean;
+  private elementBounds?: DOMRect;
+  private crosshairCursorDisposable?: Disposable;
+  private waitCursorDisposable?: Disposable;
 
   public constructor(private controller: VolumeIntersectionQueryController) {
     this.handleDragBegin = this.handleDragBegin.bind(this);
     this.handleDrag = this.handleDrag.bind(this);
     this.handleDragEnd = this.handleDragEnd.bind(this);
+
+    this.enabled = true;
   }
 
-  public initialize(element: HTMLElement): void {
+  public initialize(element: HTMLElement, api: InteractionApi): void {
     this.element = element;
+    this.api = api;
 
     this.element.addEventListener('pointerdown', this.handleDragBegin);
+    this.addCrosshairCursor();
   }
 
   public dispose(): void {
     this.element?.removeEventListener('pointerdown', this.handleDragBegin);
     window.removeEventListener('pointermove', this.handleDrag);
     window.removeEventListener('pointerup', this.handleDragEnd);
+    this.crosshairCursorDisposable?.dispose();
+    this.waitCursorDisposable?.dispose();
+
+    this.element = undefined;
+    this.api = undefined;
+    this.enabled = true;
+  }
+
+  public disable(): void {
+    this.enabled = false;
+  }
+
+  public enable(): void {
+    this.enabled = true;
   }
 
   private handleDragBegin(event: PointerEvent): void {
-    if (event.buttons === 1 && !this.isInteracting) {
+    if (this.enabled && event.buttons === 1 && !this.isInteracting) {
```

Comment:
💡 Just an idea here, I often forget what all of the button numbers map to. Do you think it'd be a good idea to add some utils around the PointerEvent buttons to make it more readable? i.e. 

```ts
private function isPrimaryClick(event: PointerEvent): boolean {
  return event.buttons === 1;
}
```
Just an idea to make some of the code more readable. Not that important.